src/appleseed/CMakeLists.txt

```diff
@@ -103,16 +103,13 @@ set (foundation_image_sources
     foundation/image/genericprogressiveimagefilereader.h
     foundation/image/icanvas.h
     foundation/image/iimagefilereader.h
-    foundation/image/iimagefilewriter.h
```

Comment:
Why did you remove this file? We want to keep the `IImageFileWriter` interface.

Reply:
Done